pkgs/cli/src/node.zig

```diff
@@ -726,35 +728,35 @@ fn constructENRFromFields(allocator: std.mem.Allocator, private_key: []const u8,
     return enr;
 }
 
-// TODO: Enable and update this test - YAML parsing for public keys is now implemented, but test expectations may need adjustment
-// test "config yaml parsing" {
-//     var config1 = try utils_lib.loadFromYAMLFile(std.testing.allocator, "pkgs/cli/test/fixtures/config.yaml");
-//     defer config1.deinit(std.testing.allocator);
-//     const genesis_spec = try configs.genesisConfigFromYAML(config1, null);
-//     try std.testing.expectEqual(9, genesis_spec.num_validators);
-//     try std.testing.expectEqual(1704085200, genesis_spec.genesis_time);
-//
-//     var config2 = try utils_lib.loadFromYAMLFile(std.testing.allocator, "pkgs/cli/test/fixtures/validators.yaml");
-//     defer config2.deinit(std.testing.allocator);
-//     const validator_indices = try validatorIndicesFromYAML(std.testing.allocator, "zeam_0", config2);
-//     defer std.testing.allocator.free(validator_indices);
-//     try std.testing.expectEqual(3, validator_indices.len);
-//     try std.testing.expectEqual(1, validator_indices[0]);
-//     try std.testing.expectEqual(4, validator_indices[1]);
-//     try std.testing.expectEqual(7, validator_indices[2]);
-//
-//     var config3 = try utils_lib.loadFromYAMLFile(std.testing.allocator, "pkgs/cli/test/fixtures/nodes.yaml");
-//     defer config3.deinit(std.testing.allocator);
-//     const nodes = try nodesFromYAML(std.testing.allocator, config3);
-//     defer {
-//         for (nodes) |node| std.testing.allocator.free(node);
-//         std.testing.allocator.free(nodes);
-//     }
-//     try std.testing.expectEqual(3, nodes.len);
-//     try std.testing.expectEqualStrings("enr:-IW4QA0pljjdLfxS_EyUxNAxJSoGCwmOVNJauYWsTiYHyWG5Bky-7yCEktSvu_w-PWUrmzbc8vYL_Mx5pgsAix2OfOMBgmlkgnY0gmlwhKwUAAGEcXVpY4IfkIlzZWNwMjU2azGhA6mw8mfwe-3TpjMMSk7GHe3cURhOn9-ufyAqy40wEyui", nodes[0]);
-//     try std.testing.expectEqualStrings("enr:-IW4QNx7F6OKXCmx9igmSwOAOdUEiQ9Et73HNygWV1BbuFgkXZLMslJVgpLYmKAzBF-AO0qJYq40TtqvtFkfeh2jzqYBgmlkgnY0gmlwhKwUAAKEcXVpY4IfkIlzZWNwMjU2azGhA2hqUIfSG58w4lGPMiPp9llh1pjFuoSRUuoHmwNdHELw", nodes[1]);
-//     try std.testing.expectEqualStrings("enr:-IW4QOh370UNQipE8qYlVRK3MpT7I0hcOmrTgLO9agIxuPS2B485Se8LTQZ4Rhgo6eUuEXgMAa66Wt7lRYNHQo9zk8QBgmlkgnY0gmlwhKwUAAOEcXVpY4IfkIlzZWNwMjU2azGhA7NTxgfOmGE2EQa4HhsXxFOeHdTLYIc2MEBczymm9IUN", nodes[2]);
-// }
+test "config yaml parsing" {
+    var config1 = try utils_lib.loadFromYAMLFile(std.testing.allocator, "pkgs/cli/test/fixtures/config.yaml");
```

Comment:
call this `config_file` and other `validators_file` and so on instead of config1, config2 since these names are confusing